ZonPHP/inc/load_language.php

```diff
@@ -55,49 +50,55 @@
         }
         $_SESSION['txt'] = $txt;
     }
-
     if (isset($_SESSION['sestaal'])) {
         $taal = $_SESSION['sestaal'];
     }
 }
-
 // date_default_timezone_set('Europe/Brussels');
 if ($taal == "nl") {
-    setlocale(LC_TIME, 'Dutch_Netherlands', 'Dutch', 'nl_NL', 'nl', 'nl_NL.ISO8859-1', 'nl_NL.UTF-8', 'nld_nld', 'nld', 'nld_NLD', 'NL_nl');
+    //commented out setlocale. Do we need it?
+    //setlocale(LC_TIME, 'Dutch_Netherlands', 'Dutch', 'nl_NL', 'nl', 'nl_NL.ISO8859-1', 'nl_NL.UTF-8', 'nld_nld', 'nld', 'nld_NLD', 'NL_nl');
```

Comment:
Maybe just remove the comment lines, we will have the history in git... so we can get rid of some code